examples/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 [dev-dependencies]
 anyhow = "1.0"
 tokio = { version = "1.10.0", features = ["full"] }
-workspaces = { version = "0.1.0", path = "../workspaces" }
+workspaces = { version = "0.2.0", path = "../workspaces" }
```

Comment:
fair enough, I was just upping the version just in case some people out there were already making use of the API as-is, but I did add a warning that things would change on the README. I don't feel strongly at all, so I can just keep it version-agnostic then